data/functions.sql

```diff
@@ -83,24 +83,33 @@ BEGIN
 END;
 $$ LANGUAGE plpgsql IMMUTABLE;
 
--- mz_get_rel_network returns a network tag, or NULL, for a
--- given way ID.
+-- mz_get_rel_networks returns a list of triples of route type,
+-- network and ref tags, or NULL, for a given way ID.
 --
 -- it does this by joining onto the relations slim table, so it
 -- won't work if you dropped the slim tables, or didn't use slim
 -- mode in osm2pgsql.
 --
-CREATE OR REPLACE FUNCTION mz_get_rel_network(
+CREATE OR REPLACE FUNCTION mz_get_rel_networks(
   way_id bigint)
 RETURNS text AS $$
```

Comment:
Is it easy to have this return back an array, hstore, or json with the values instead of a delimited string? This is just meant for the python transform to consume and process right?